speculation-rules/prerender/resources/utils.js

```diff
@@ -234,7 +234,7 @@ function test_prerender_restricted(fn, expected, label) {
   }, label);
 }
 
-function test_prerender_defer(fn, label) {
+function test_prerender_defer(fn, label, timeout = delay) {
```

Comment:
Where is this `delay` variable coming from? Why is the `timeout` parameter defined here unused?

Reply:
Crud, removed.